src/util/xeval/evaluator.rs

```diff
@@ -20,14 +20,15 @@ use std::cmp::Ordering;
 use tipb::expression::{Expr, ExprType};
 
 /// `Evaluator` evaluates `tipb::Expr`.
+#[derive(Default)]
 pub struct Evaluator {
     // column_id -> column_value
     row: HashMap<i64, Datum>,
 }
 
 impl Evaluator {
-    pub fn new(row: HashMap<i64, Datum>) -> Evaluator {
-        Evaluator { row: row }
+    pub fn insert(&mut self, col_id: i64, datum: Datum) {
```

Comment:
`Insert` is a little confused, for it is usually used in insert a row data, but here is only one column, so how about `set` or other else instead?